test/common/chromium-cdp-driver.js

```diff
@@ -31,6 +31,7 @@
 const CDP = require('chrome-remote-interface');
 
 let enable_debug = false;
+const show_duplicate_messages = true;
```

Comment:
Overengineered, IMHO. If this works and an inspection of the raw log doesn't show us a gazillion duplicates, then let's just drop it for good. For the "we need it again" case there is `git revert` :grin:

Reply:
Can't nobody find that in the commit history... :-) But yes, I am happy to rip it out completely.